cardano-testnet/src/Testnet/Components/SPO.hs

```diff
@@ -401,3 +405,45 @@ registerSingleSpo identifier tap@(TmpAbsolutePath tempAbsPath') nodeConfigFile s
               poolColdVkeyFp
               currentRegistedPoolsJson
   return (poolId, poolColdSkeyFp, poolColdVkeyFp, vrfSkeyFp, vrfVkeyFp)
+
+
+-- | Generates Stake Pool Operator (SPO) voting files (without signing)
```

Comment:
Actually, this may be a mistake, because voting files are not meant to be signed, only transactions can. I will remove the comment in this case instead. In the case of `createVotingTxBody`, I will explain further what it means